draft-ietf-ppm-dap.md

```diff
@@ -114,6 +114,25 @@ seen in the clear by any server.
 
 (\*) Indicates a change that breaks wire compatibility with the previous draft.
 
+04:
+
+- Introduce resource oriented HTTP API. (#278, #398, #400) (\*)
```

Comment:
I would add a sentence or explaining what "resource oriented" means and the motivation for this change.

Reply:
There's ample discussion of that in the referenced issue and PRs. I don't think it's productive to re-hash it here.